submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/FtpResource.kt

```diff
@@ -0,0 +1,19 @@
+package ac.uk.ebi.biostd.submission.web.resources
+
+import ac.uk.ebi.biostd.persistence.common.filesystem.FtpFilesService
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * This resource is just temporary to generate the FTP links
```

Comment:
The content should be the same as in [this other PR](https://github.com/EBIBioStudies/biostudies-backend-services/pull/303/files#diff-e3ac44e3f967c9258f3a7188b9382c50be834218fc2c6ed29973e9c2e0e415c7R10) in order to avoid conflicts when promoting to PROD